README.md

```diff
@@ -7,11 +7,38 @@ The PyFingerprint library allows to use ZhianTec ZFM-20, ZFM-60, ZFM-70 and ZFM-
 **Note:** The library is inspired by the C++ library from Adafruit Industries:
 <https://github.com/adafruit/Adafruit-Fingerprint-Sensor-Library>
 
-## Package building on Debian
+## Installation
+
+There are two ways of installing PyFingerprint: Installation of the **stable** or **development** version. The stable version is distributed through the PM Code Works APT repository and is fully tested but does not contain the latest changes.
+
+### Installation of the stable version
+
+Add PM Code Works repository:
+
+* Debian 9/Ubuntu 16 & 17:
+
+    `~# echo "deb http://apt.pm-codeworks.de stretch main" | tee /etc/apt/sources.list.d/pm-codeworks.list`
```

Comment:
Why this is put in code braces? Please use consistently "sudo" or the `$` and `#` style. But for beginner user i would suggest "sudo" because they know it and more advanced users can change it for they own if they want. And lets remove the `~` because it does not add any information.

Reply:
Code braces are essentially in equidations. Yes, you are right: We should better use `sudo` as it is beginner friendly. But then we should also remove the `#` as this might look like a comment.